src/wallet/mod.rs

```diff
@@ -652,21 +669,36 @@ where
         details.sent = selected_amount;
 
         let mut fee_amount = fee_amount.ceil() as u64;
+        if builder.has_absolute_fee() {
+            fee_amount = match builder.fee_policy.as_ref().unwrap() {
+                FeePolicy::FeeAmount(amount) => *amount,
+                _ => fee_amount,
+            }
+        };
```

Comment:
You probably won't need to re-override the amount if you set the `fee_rate` correctly to `0.0`.